docker/jenkins/Dockerfile.windows

```diff
@@ -20,7 +20,7 @@ RUN $env:chocolateyUseWindowsCompression = 'true'; `
 RUN choco install -y cmake --installargs 'ADD_CMAKE_TO_PATH=""System""' --fail-on-error-output; `
   choco install -y jdk8; `
   choco install -y -i ant; `
-  choco install -y windows-sdk-10.1 --version 10.1.17134.12; `
+  choco install -y windows-sdk-10.1 --version 10.1.18362.1; `
```

Comment:
Is it expected that the versions don't match with e.g. https://github.com/rstudio/rstudio/pull/10746/files?w=1#diff-0153cde0289d672e5c94275aceb9ae1a23df29e044a174d7ea567cc8c4822fd4R128?

`-winsdk=10.0.19041.0`